src/Metadata/Extractor/XmlResourceExtractor.php

```diff
@@ -450,4 +453,23 @@ private function buildGraphQlOperations(\SimpleXMLElement $resource, array $root
 
         return $data;
     }
+
+    private function buildPersistenceMeans(\SimpleXMLElement $resource): ?PersistenceMeansInterface
+    {
+        $persistenceMeans = $resource->persistenceMeans ?? null;
+        if (!isset($persistenceMeans)) {
```

Comment:
```suggestion
        if ($persistenceMeans ?? false) {
```

Reply:
When I apply this change, I have a phpstorm bug with this message : `[EA] Resolved operands types are not complimentary, while they should be ([\SimpleXMLElement] vs [bool]).`

Reply:
You can either cast the expression with `(bool)` or just ignore this message. Using `null` instead of `false` can maybe fix the message (and is probably better).

Reply:
`isset` was a remnant from before `$persistenceMeans` assignation :grimacing: 

```php
if (!$persistenceMeans) {
```

should be enough.